redhat/patches/0001-dockerfile.patch

```diff
@@ -15,8 +15,18 @@ index 5b6f43b..954235d 100644
  RUN CGO_ENABLED=1 go build -gcflags "all=-N -l" -o server_debug main.go
 
  # Multi-Stage production build
--FROM golang:1.20.7@sha256:bc5f0b5e43282627279fe5262ae275fecb3d2eae3b33977a7fd200c7a760d6f1 as deploy
+-FROM golang:1.21.1@sha256:c416ceeec1cdf037b80baef1ccb402c230ab83a9134b34c0902c542eb4539c82 as deploy
 +FROM registry.access.redhat.com/ubi9/go-toolset@sha256:52ab391730a63945f61d93e8c913db4cc7a96f200de909cd525e2632055d9fa6 as deploy
 
  # Retrieve the binary from the previous stage
  COPY --from=builder /opt/app-root/src/server /usr/local/bin/fulcio-server
+@@ -41,3 +41,9 @@ RUN go install github.com/go-delve/delve/cmd/dlv@v1.8.0
+
+ # overwrite server and include debugger
+ COPY --from=builder /opt/app-root/src/server_debug /usr/local/bin/fulcio-server
```

Comment:
I've been trying to figure out why these lines are here - it looks like they're not actually newly added, but are just the way the diff is displaying, that differs from how it was displaying before. I could use another OK on this.

Reply:
This does seem incorrect. These lines already exist in the target Dockerfile. E.g. [here](https://github.com/sigstore/fulcio/blob/51f09e89619130b724aa2dd0b10814a1e0fe4e12/Dockerfile#L43). You should not have to add it in this diff. Have you tried seeing if the diff applies correctly to the upstream Dockerfile on sigstore/fulcio at `main`?

Reply:
Looking at diffs of diffs is crazy making!